coursebuilder/views/unit.html

```diff
@@ -130,7 +130,7 @@
                 {% if next_button_url %}
                   <a href="{{ next_button_url }}" class="btn right">{{ gettext('Next') }}</a>
                 {% else %}
-                  <a href="course" class="btn right"> {{ gettext('End') }} </a>
+                  <a href="/#!explorer" class="btn right"> {{ gettext('End') }} </a>
```

Comment:
@Tawakalt Have you tested this?

Reply:
@Tawakalt this is only supposed to change for the last unit of a course.

Reply:
@chegejames Yes I have. The END button appears only in the last unit of a course.

Reply:
@Tawakalt it also appears in the last lesson of a unit e.g https://cfafrica-mooc-dev.appspot.com/test/unit?unit=1&lesson=6

Reply:
@chegejames Didn't see that. Will fix it.